shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java

```diff
@@ -71,13 +83,38 @@ public void clearClient() {
                 clearTextInputClient();
             }
         });
+        this.platformViewsResolver = platformViewsResolver;
     }
 
     @NonNull
     public InputMethodManager getInputMethodManager() {
         return mImm;
     }
 
+    /***
+     * Use the current platform view input connection until unlockPlatformViewInputConnection is called.
+     *
+     * The current input connection instance is cached and any following call to @{link createInputConnection}.
```

Comment:
Looks like there might be a typo or something here? Should the sentence end with an action description?

Reply:
fixed